src/activity_queue/mod.rs

```diff
@@ -0,0 +1,247 @@
+//! Queue for signing and sending outgoing activities with retry
+//!
+#![doc = include_str!("../../docs/09_sending_activities.md")]
+
+use crate::{
+    config::Data,
+    traits::{ActivityHandler, Actor},
+};
+use anyhow::anyhow;
+
+use bytes::Bytes;
+use itertools::Itertools;
+use openssl::pkey::{PKey, Private};
+use reqwest_middleware::ClientWithMiddleware;
+use serde::Serialize;
+use std::{fmt::Debug, sync::atomic::Ordering, time::Duration};
+
+use tracing::{debug, info, warn};
+use url::Url;
+
+use self::{request::sign_and_send, retry_queue::RetryQueue};
```

Comment:
There should be no newlines between imports, so that they get formatted and reordered properly.